src/backend/context.ts

```diff
@@ -1,10 +1,51 @@
 import { CreateNextContextOptions } from "@trpc/server/adapters/next";
 import { getDb } from "backend/db/lib/get-db";
+import { Session, getServerSession } from "next-auth";
 import { Env } from "./lib/types";
+import { getNextAuthOptions } from "./auth/options";
 
-export const createContext = (options: CreateNextContextOptions) => {
+type Auth =
+  | {
```

Comment:
OOC, what is the leading pipe on this line doing - adding null or undefined to `Auth` or it's syntactical sugar to line up the pipe on line 11?

Reply:
Put there by Prettier to line up the pipes 👍